src/test/resources/tf/GCP-IAM-user-have-overly-permissive-KMS-role/GCP-IAM-user-have-overly-permissive-KMS-role-negative.json

```diff
@@ -0,0 +1,92 @@
+{
+  "locals": [
+    {
+      "test_description": "No relevant resource/data"
+    }
+  ],
+  "not_data": [
+    {
+      "google_iam_policy": [
+        {
+          "x": [
+            {
+              "binding": [
+                {
+                  "members":  ["user:c@d.com"],
+                  "role": "roles/cloudkms.admin"
+                }
+              ]
+            }
+          ]
+        },
+        {
+          "y": [
+            {
+              "binding": [
+                {
+                  "member":  "user:a@b.com",
+                  "role": "roles/cloudkms.admin"
```

Comment:
I guess this file should be a positive test case as it will match the rule.

Reply:
Sorry for the confusion. The top element is "not_data", so won't match.

Reply:
oh. sorry i missed that.